### PR TITLE
docs: Windows add Antivirus section

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
With this PR we document the fact that `bareos processes` should be excluded from any kind of on access scanner.

#OP 5474

#### Please check

- [X] Short description and the purpose of this PR is present _above this paragraph_
- [X] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Check backport line
- [x] Required backport PRs have been created

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

File: docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst [hunk 1]
```diff
@@ -474,6 +474,20 @@ If you want to see if the File daemon has properly opened the port and is listen
 
    netstat -an | findstr 910[123]
 
+Windows Antivirus
+-----------------
+
+.. index:: Antivirus; Windows
+.. index:: Windows; Antivirus
+
+The Windows builtin anti-virus **Defender** is enabled by default. If you care about performance, it is advised to add the :command:`bareos-dir.exe`, :command:`bareos-sd.exe`, :command:`bareos-fd.exe` to the list of excluded monitored processes. Otherwise you may suffer by i/o throughput and excessive cpu consumption.
```
Comment (pstorz) [inline docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst:483]:
```suggestion
The Windows builtin antivirus **Defender** is enabled by default and will scan every single file being backed up during the backup process. This leads to an extremely slow throughput with a very high cpu load.

Therefore it is advised to add the :command:`bareos-fd.exe` to the list of excluded monitored processes on computers being backed up via the bareos filedaemon.
On systems running the director or storage daemon on windows, also add the processes
:command:`bareos-dir.exe`, :command:`bareos-sd.exe`
```

File: docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst [hunk 2]
```diff
@@ -474,6 +474,20 @@ If you want to see if the File daemon has properly opened the port and is listen
...
+
+.. image:: /include/images/windows-defender-exclusion.png
```
Comment (pstorz) [inline docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst:485]: The picture is very blurry. Maybe that can be improved?

File: docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst [hunk 3]
```diff
@@ -474,6 +474,20 @@ If you want to see if the File daemon has properly opened the port and is listen
...
+   :width: 80.0%
+
+The same remark can be done to any other similar on-access anti-virus scanner. Please also check next section about possible troubles with antivirus on Windows.
```
Comment (pstorz) [inline docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst:488]:
```suggestion
The same remark can be done to any other similar on-access antivirus scanner. Please also check next section about possible troubles with antivirus on Windows.
```

**Review by pstorz — CHANGES_REQUESTED** [review]

File: docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst (hunk 2 above)
Comment (bruno-at-bareos) [reply docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst:485]: I will retry another one. seems compression and resize of this one destroyed the quality

Comment (pstorz) [reply docs/manuals/source/TasksAndConcepts/TheWindowsVersionOfBareos.rst:489]: Looks much better now!

**Pull request merged**